include/wx/textctrl.h

```diff
@@ -608,6 +608,69 @@ class WXDLLIMPEXP_CORE wxTextAttr
     wxString            m_listStyleName;
 };
 
+// ----------------------------------------------------------------------------
+// Search features for wxTextCtrl
+// ----------------------------------------------------------------------------
+
+enum class wxTextSearchDirection
+{
+    Down,
+    Up
+};
+
+// search options
+// --------------
+struct wxTextSearch
+{
+    wxTextSearch(const wxString& text) : m_searchValue(text) {}
+
+    wxTextSearch& SearchValue(const wxString& value)
+    {
+        m_searchValue = value;
+        return *this;
+    }
+
+    wxTextSearch& MatchCase(const bool matchCase = true)
```

Comment:
We don't use [useless top level const](https://clang.llvm.org/extra/clang-tidy/checks/readability/avoid-const-params-in-decls.html):

```suggestion
    wxTextSearch& MatchCase(bool matchCase = true)
```

(also below)